pkgs/by-name/mi/microsocks/package.nix

```
@@ -0,0 +1,28 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  lib
+}:
+stdenv.mkDerivation rec {
+  pname = "microsocks";
+  version = "1.0.4";
+  src = fetchFromGitHub {
+    owner = "rofl0r";
+    repo = "microsocks";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    sha256 = "sha256-cB2XMWjoZ1zLAmAfl/nqjdOyBDKZ+xtlEmqsZxjnFn0=";
```

It's preferred to write `hash` when specifying the hash value.

```suggestion
    hash = "sha256-cB2XMWjoZ1zLAmAfl/nqjdOyBDKZ+xtlEmqsZxjnFn0=";
```